api/src/exam-environment/schemas/exam-generated-exam.ts

```diff
@@ -0,0 +1,23 @@
+import { Type } from '@fastify/type-provider-typebox';
+import { STANDARD_ERROR } from '../utils/errors';
+
+export const examEnvironmentPostExamGeneratedExam = {
+  body: Type.Object({
+    examId: Type.String()
+  }),
+  headers: Type.Object({
+    'exam-environment-authorization-token': Type.String()
+  }),
+  response: {
+    200: Type.Object({
+      data: Type.Object({
+        exam: Type.Record(Type.String(), Type.Unknown()),
+        examAttempt: Type.Record(Type.String(), Type.Unknown())
```

Comment:
TODO: not `unknown`?

Reply:
Yes, definitely would be nice to not be an unknown. That does require some people not requesting schema changes too often - once everything is finalized